### DISCUSSION TITLE
Active Directory authentication?

### DISCUSSION THREAD

**Discussion by dreniarb** [Budibase/budibase#5039] [Help] [answered]
Can someone point me in the right direction to get Active Directory authentication working? This news release mentions it specifically and even shows a picture that states "Sign in with Active Directory" but I'm not sure how to do it.

https://budibase.com/blog/july-2021-release/

Thanks

**Comment by shogunpurple** [comment, marked as answer]
Hi @dreniarb 
The docs here will provide more detail - https://docs.budibase.com/docs/sso-with-azure-ad

Thanks!

**Reply by dreniarb** [reply]
Thanks for the quick reply.  Follow-up question: 

We are on premise only.  Will this work with a non-Azure AD infrastructure?
